src/Particular.LicensingComponent.UnitTests/AuditQuery_Tests.cs

```diff
@@ -32,14 +35,15 @@ public async Task Should_return_known_endpoints_if_any()
 
         //Act
         var endpoints = await auditQuery.GetKnownEndpoints(default);
+        var serviceControlEndpoints = endpoints.ToList();
```

Comment:
I think this name is meaningless. I would much rather keep endpoints and change it to something like

        var endpoints = (await auditQuery.GetKnownEndpoints(default)).ToList();